_sass/components/_list.scss

```diff
@@ -27,41 +27,32 @@
   background-size: 1rem;
 }
 
-.help-article > h2:first-of-type,
-.list-styled {
+.help-article > h2:first-of-type {
   margin-top: 2rem;
+}
 
-  & + ul, 
-  & + ol {
-    list-style-type: none;
-    margin-left: 0;
-    padding-left: 0;
+.help-article > h2:first-of-type + ul,
+.help-question-list,
+.page-content__prose ul.help-question-list {
+  list-style-type: none;
+  margin-left: 0;
+  padding-left: 0;
 
-    li:before {
-      content: "?";
-      background-color: $navy;
-      padding: 2px 8px;
+  li {
+    @include u-margin-y(1);
+
+    &::before {
+      background-color: color("primary-darker");
       border-radius: 50%;
-      color: $white;
-      line-height: 2.2;
-      margin-right: 0.75rem;
+      color: color("white");
+      content: "?";
+      display: inline-block;
       font-weight: bold;
-    }
-
-    a {
-      color: $blue-mid;
-    }
-  }
-  
-  & + ol {
-    counter-reset: num;
-
-    li {
-      counter-increment: num;
-    }
-
-    li:before {
-      content: counter(num);
+      height: 25px;
+      line-height: 25px;
+      margin-right: .75rem;
+      text-align: center;
+      width: 25px;
```

Comment:
Looks like you removed the styling for ordered lists? Do you have an alternate approach for achieving the following style?
![image](https://user-images.githubusercontent.com/381122/102911902-b4694a80-444a-11eb-9c0b-2468873bad42.png)

Reply:
Can you add a `.page-content__prose ol` list so we can keep this style?

Reply:
> Looks like you removed the styling for ordered lists? Do you have an alternate approach for achieving the following style?

Ah, I think I just hadn't seen this context and didn't see any current usage in the site. I can restore the ordered list styles.

Can you point me to the design associated with this list style?

Reply:
> Can you point me to the design associated with this list style?

I found it in the [Sketch file](https://github.com/18F/identity-design-assets/blob/master/Projects/Outreach/Public%20Site%20-%20Brochure/team-katherine-brochure-v2.sketch).

Reply:
Restored ordered list styling in 98a7072.

Reply:
Not sure if it was expected that the styles take effect on help pages like "Forgot your password" ([current demo site link](https://demo.login.gov/help/trouble-signing-in/forgot-your-password/)). I assume they are? They are now after these changes. It appears to work pretty well, with one exception: Multi-paragraph step texts aren't inlined to the list item:

![image](https://user-images.githubusercontent.com/1779930/102928348-872a9580-4466-11eb-9961-a62703392a9f.png)

This also happens with the previous implementation as well.

I think there might be an option using `padding-left` on the content and `position: absolute` on the styled step indicator. I'll tinker with it a bit, but if it proves to be too much work, might defer to a follow-up pull request.

Reply:
> I think there might be an option using `padding-left` on the content and `position: absolute` on the styled step indicator. I'll tinker with it a bit, but if it proves to be too much work, might defer to a follow-up pull request.

Updated in 1a15f8f.

Ordered|Unordered
---|---
![ordered list](https://user-images.githubusercontent.com/1779930/102930494-8267e080-446a-11eb-8e8d-22f324c1b8aa.png)|![unordered list](https://user-images.githubusercontent.com/1779930/102930501-87c52b00-446a-11eb-8c50-e0b0de97019a.png)